App/Repositories/IRepository.cs

```diff
@@ -2,14 +2,15 @@
 
 namespace App.Repositories
 {
-    interface IRepository
+    public interface IRepository
     {
         long AddTask(Task newTask);
-        void DeleteTask(long task_id);
-        void UpdateTaskTitle(long task_id, string newTitle);
-        void UpdateTaskDescription(long task_id, string newDescription);
-        void UpdateTaskDueTime(long task_id, DateTimeOffset newDue);
-        Task GetTask(long task_id);
+        void DeleteTask(long taskId);
+        void CompleteTask(long taskId);
```

Comment:
I woner if `CompleteTask` should be changed to `void UpdateTaskStatus(long task_id, bool isDone)`, to be consistent with other names of the kind, and to allow "un-doing" a task.

Reply:
As I said in issue #41, at this point we needed to add a method that changes is_done to true (and later to false). I think I'll do that now.
By the way, should we throw an exception if we change the is_done value back to what it was (we don't expect the completed task to be completed again)?

Reply:
I think an exception would be the wrong kind of response. We could instead make `CompleteTask`, or `UpdateTaskStatus`, to return the previous value of the field.

Reply:
Ok, then I'll remove the exception throwing for now, and then we'll change it as needed